Доступная разметка главной страницы

okmeow opened the pull request:
---
:mortar_board: [Доступная разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/1615961/tasks/13)

okmeow committed:
Стилизует стрелки у select

okmeow committed:
Делает главную страницу по Pixel Perfect

okmeow committed:
Прописывается псевдокласс disabled для инпутов

adaptedbee commented on index.html:
```diff
@@ -63,71 +64,71 @@ <h2 class="visually-hidden">Преимущества</h2>
             <input class="newsletter-field"  placeholder="Ваш e-mail" type="email" name="newsletter-email" id="newsletter-email" required>
             <button class="button newsletter-button" type="submit">
-              Подписаться
+              <span class="newsletter-button-text">Подписаться</span>
```
span не нужен, стили newsletter-button-text можно самой кнопке задать

adaptedbee commented on styles/styles.css:
```diff
@@ -1611,6 +1640,11 @@ input[type=number]::-webkit-outer-spin-button {
+  overflow: hidden;
+  background-image: url('../images/icons/filter-arrow.svg');
+  background-repeat: no-repeat;
+  background-position: 254px center;
```
Вот эти 254px смущают - если элемент станет шире, стили сломаются
Может, стоит в процентах задавать или псевдоэлементом привязывать к правому краю

adaptedbee commented on styles/styles.css:
```diff
@@ -1979,6 +2013,11 @@ input[type=number]::-webkit-outer-spin-button {
+  overflow: hidden;
+  background-image: url('../images/icons/filter-arrow.svg');
+  background-repeat: no-repeat;
+  background-position: 127px center;
```
И здесь

okmeow committed:
Исправляет ошибки

А также делает, чтобы при переполнении контента размытые фоновые изображения тянулись по всему контейнеру
